### ISSUE TITLE
BC Racer overhead map graphics glitch 

### ISSUE THREAD

**Issue by carlo3do** [MiSTer-devel/S32X_MiSTer#26]
BC Racer overhead map graphics glitch on first level second level seem fine did not tried the rest of levels

**Comment by a-barone** [comment]
Hi!

I thought it would be helpful if I provided a proper description of the bug reported above.

Scenario:
- The RACE DIFFICULTY has to be set to Easy (which is the default setting) and it has to be the first race/track in order to reproduce this bug, like this:
![image](https://user-images.githubusercontent.com/19418951/173722790-e9c4cb9e-3f4b-4b3d-807c-3198e90451a0.png)

- Then once you start the race, you'll see this:
![image](https://user-images.githubusercontent.com/19418951/173723038-3879c0b5-e94f-49af-b89d-88f6f265d207.png)

- Be aware that for each RACE DIFFICULTY setting each stage has a different circuit (they increase in complex as you set a harder difficulty) and thus a different overhead map. It seems that this issue is specific to this one overhead map which is unique to the Easy track of the first race in the game. 
In the video below, I tried the other race difficulty settings for the same stage:
Real Hardware (Left) |  S32X Core (Right)
https://youtu.be/T7Kc0UOjgr4
